docs/users-guide/14-x-rays.md

```diff
@@ -44,3 +46,25 @@ The comparison report shows how many rows there are in the segment versus the pa
 ![Comparison report](images/x-ray-comparison.png)
 
 An example for where this can be especially useful is a scenario where you've defined many different segments for your users or customers, like "Repeat Customers," "Users between 18 and 35," or "Female customers in Kalamazoo who dislike cheese." You can open up the x-ray for any of these segments, and then compare them to the larger Users or Customers table to see if there are any interesting patterns or differences.
+
+## Automated insights
+Metabase hasn't quite achieved self-awareness, but it has gotten smarter recently. It will now show you relevant insights about your data at the top of x-rays about time series or numeric fields, provided there's something insightful to say.
+
+![Insights](./images/insights.png)
+
+Here's some of the things Metabase might point out to you:
+- The normal range of values in your data.
+- Whether or not there seem to be a lot of missing values or zeros in your data.
+- Whether or not your data has an overall trend, and if so, what kind of trend.
+- Whether or not your data can be broken up into different distinct stages of growth or decline.
+- Whether or not your data is stationary.
+- Whether or not your data has a strong or mild autocorrelation [@sb, I don't totally understand this one. Something to do with lagged repeating patterns?]
```

Comment:
Should probably @sbelak here for help.

Reply:
Yep, thanks for catching that. @sbelak, there were a couple of statistical terms in here that I couldn't explain easily from context or light Wikipedia-ing, so your input here would be delightful.

Reply:
Not sure we need to specify these out entirely. I expect them to change fairly often, and this feels a bit too specific, and nerdy. If anything we should be changing the insights copy to english vs using statistical terms.